migrations/20151021_usernames_emails_lowercase.js

```diff
@@ -39,7 +39,7 @@ function findUsers(gt){
       console.log('User: ', countUsers, user._id);
 
       var update = {
-        $set: {};
+        $set: {}
```

Comment:
this should be a comma not a semicolon that's not allowed in objects

Reply:
This PR *removes* the errorneous semicolon.

`$set` is the only property in this object literal, so replacing it with a comma would be another mistake:
```
var update = {
    $set: {},
};
```

Reply:
@esbena oh sorry, I completely mixed the added and remove code :)